extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -61,17 +61,19 @@ import {layoutRectLtwh, layoutSizeFromRect} from '../../../../src/layout-rect';
 import {macroTask} from '../../../../testing/yield';
 import {resetScheduledElementForTesting} from '../../../../src/service/custom-element-registry';
 import {data as testFragments} from './testdata/test_fragments';
+import {toggleExperiment} from '../../../../src/experiments';
 import {data as validCSSAmp} from './testdata/valid_css_at_rules_amp.reserialized';
 
-// eslint-disable-next-line no-undef
 if (NO_SIGNING_RTV) {
-  describes.realWin('no signing', {amp: true}, (env) => {
+  describes.realWin('amp-a4a: no signing', {amp: true}, (env) => {
     let doc;
     let element;
     let a4a;
+    let attachShadowStub;
 
     beforeEach(() => {
       doc = env.win.document;
+      env.sandbox.stub(AmpA4A.prototype, 'isInNoSigningExp').returns(true);
```

Comment:
this is probably better than what I was doing :) with this can we just delete the `if (NO_SIGNING_RTV)` so these always run?

Reply:
Yes, we can. I was actually going to. But that'd make diff very large. So let's finish the review first and I'll cleanup before merging.

Reply:
Done.